packages/editor/src/components/rich-text/index.js

```diff
@@ -876,7 +876,6 @@ export class RichText extends Component {
 								style={ style }
 								defaultValue={ this.valueToEditableHTML( record ) }
 								isPlaceholderVisible={ isPlaceholderVisible }
-								aria-label={ placeholder }
```

Comment:
Maybe we can still use the placeholder as aria-label for cases where aria-label is not explicitly set. Some plugins may not be setting aria-label and relying on this feature.

Reply:
@jorgefilipecosta I thought about that initially, but the whole reason this ticket was created was to prevent the `placeholder` from doubling as the `aria-label`, so I opted to not leave that fallback behavior in and make the inclusion of a label more explicit, like it is for the placeholder.

If we want to go down that path I can create a new prop for the `aria-label`, check if it's present, if not, use the placeholder, it isn't a huge lift, just going back through all the files. I see pros and cons to both implementations.

@afercia would love your feedback on this as well.

Reply:
Can we leave it as is? At the moment it defaults to `placeholder` when `aria-label` is not provided, otherwise overrides the value.

Reply:
@gziolo So, the spec says, no, we shouldn't allow placeholder to be used in that way. That's how I would lean, not allowing people to do incorrect things. At the same time, I know some browsers will fallback to placeholder as the last ditch accessible name for an element.

Reply:
I probably wasn't clear enough. At the moment when no `aria-label` is provided then it's set to the value which placeholder holds. It can still be overridden by explicitly providing `aria-label` as it happens in this PR. So my question was whether it makes sense to have this backup plan in case someone doesn't set `aria-label` when using this component.

Reply:
Please see my comments in the review https://github.com/WordPress/gutenberg/pull/12473#pullrequestreview-200303118

The placeholder value as fallback wouldn't make sense. These placeholders (not real HTML placeholders) are used in a wrong way as sort of visual label but their _value_ doesn't make sense as a label. Example: the quote block:

`Write quote…`

this is the current quote block label, _whether the quote is empty or has some content_.  When it has some content, `Write quote…` clearly doesn't make sense. 

Also, would you ever label a form field "Write first name"? Certainly not. You would just use "First name".

Lastly, (see again the review), when the visible placeholder and the actual aria label mismatch, speech input users can't use these fields. Example:
```
visible text: Start writing or type / to choose a block
aria-label:   Empty block; start writing or type forward slash to choose a block
```
Users will try to voice a command based on the text they can see, and will try:
`Click Start writing or type / to choose a block`

That wouldn't work because the accessible name is:
`Empty block; start writing or type forward slash to choose a block`

Reply:
Thanks for further clarifying the behavior. I understand that using placeholder text as fallback doesn't make sense. Let's leave it as is. 

I don't quite follow the last part about `aria-label` using placeholder value. If you provide `aria-label` it always overrides `aria-label={ placeholder }` so I don't see it happening. Anyway, that's important as we'are leaving PR in the current shape.

Reply:
I'm fine with this. It seems better to let native labelling behaviour take over than forcing placeholder text as the label.